s30310-KacperKoszelak/zajecia3/3_3.txt

```diff
@@ -30,11 +30,12 @@ Zobacz co się stanie:
 W opisie rozwiązania zadania umieść:
 1. jak rozumiesz co się stało?
     -zawartość "wynik3_1.txt" nie została zapisana, a zawartość "wynik3_2.txt" 
-    została zachowana
+     została zachowana
 
 2.czym oba te przykłady się różnią?
-    -CTRL+C nie zapisuje wprowadzanego tekstu, a CTRL+D zapisuje
+    -CTRL+C natychmiastowo zabija obecnie wykonywany proces, a CTRL+D wychodzi 

```

Comment:
^D nie wychodzi z procesu, nie zabija go i nie kończy. Fakt zakończenia się procesu wynika z końca danych wchodzących a nie z jego zabicia. Program nie ma więcej nic do przetworzenia więc kończy pracę ale po przetworzeniu wpisanych danych. Ponadto program nie zapisuje nic do pliku tylko na swoje standardowe wyjście, które to dopiero przez basha w którym jest uruchomiony proces jest przekierowywane do pliku.